eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -94,18 +102,23 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
           statement.executeUpdate("CREATE INDEX network_fees_timestamp_idx ON audit.network_fees(timestamp)")
           statement.executeUpdate("CREATE INDEX channel_events_timestamp_idx ON audit.channel_events(timestamp)")
           statement.executeUpdate("CREATE INDEX channel_errors_timestamp_idx ON audit.channel_errors(timestamp)")
-        case Some(v@4) =>
-          logger.warn(s"migrating db $DB_NAME, found version=$v current=$CURRENT_VERSION")
-          migration45(statement)
-          migration56(statement)
-          migration67(statement)
-        case Some(v@5) =>
-          logger.warn(s"migrating db $DB_NAME, found version=$v current=$CURRENT_VERSION")
-          migration56(statement)
-          migration67(statement)
-        case Some(v@6) =>
+          statement.executeUpdate("CREATE INDEX channel_updates_cid_idx ON audit.channel_updates(channel_id)")
+          statement.executeUpdate("CREATE INDEX channel_updates_nid_idx ON audit.channel_updates(node_id)")
+          statement.executeUpdate("CREATE INDEX channel_updates_timestamp_idx ON audit.channel_updates(timestamp)")
+        case Some(v@(4 | 5 | 6 | 7)) =>
           logger.warn(s"migrating db $DB_NAME, found version=$v current=$CURRENT_VERSION")
-          migration67(statement)
+          if (v < 5) {
+            migration45(statement)
+          }
+          if (v < 6) {
+            migration56(statement)
+          }
+          if (v < 7) {
+            migration67(statement)
+          }
+          if (v < 8) {
+            migration78(statement)
+          }
```

Comment:
Totally understand the need to improve this, but let's do that in a separate PR and fix all db files at once.

Otherwise LGTM

Reply:
reverted that part